runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -180,16 +180,8 @@ public void onScriptEvent(ScriptCallbackEvent event)
 					return;
 				}
 
-				boolean tagSearch = searchInput.startsWith(TAG_SEARCH);
-				String search;
-				if (tagSearch)
-				{
-					search = searchInput.substring(TAG_SEARCH.length()).trim();
-				}
-				else
-				{
-					search = searchInput;
-				}
+				boolean tagSearch = searchInput.matches(TAG_SEARCH);
+				String search = tagSearch ? searchInput.replaceAll(TAG_SEARCH, "$1").trim() : searchInput;
```

Comment:
I don't think this should be a ternary for clarity's sake.

If anything, the search section should be moved up to here.

Reply:
done